src/Uno.UWP/UI/Text/FontWeight.skia.cs

```diff
@@ -0,0 +1,15 @@
+﻿using SkiaSharp;
+
+namespace Windows.UI.Text
+{
+	partial struct FontWeight
+	{
+		public SKFontStyleWeight ToSkiaWeight()
+		{
+			// Uno weight values are using the same system,
+			// so we can convert directly to Skia system
+			// without need for a mapping.
+			return (SKFontStyleWeight)this.Weight;
```

Comment:
If I'm right `Weight` is just an int so the value might be an invalid `SKFontStyleWeight`

Reply:
Nope.  Because scalar values are valid too.  So that's correct.